pkg/utils/asynciter.go

```diff
@@ -167,6 +167,7 @@ func (ic *AsyncItemsCursor) NextItem() (v3io.Item, error) {
 	}
 
 	getItemsResp := resp.Output.(*v3io.GetItemsOutput)
+	resp.Output = nil
```

Comment:
Not sure it is necessary, look at `asynciter.go:156` it releases the responses on defer

Reply:
Response.Release() only returns the fastHttpResponse body back to the pool, it doesn't release the base64 decoded blobs

Reply:
This line does nothing :)

Reply:
Ok, I found the actual leak. It's the `defer resp.Release()`, which is inappropriate in case of recursion, and causes both the fasthttp response and the wrapper to be preserved until the recursion is done. Instead, it should be released __before__ the recursive call.

Reply:
https://github.com/v3io/v3io-tsdb/pull/284